presto-prometheus/src/main/java/io/prestosql/plugin/prometheus/PrometheusTimeSeriesValue.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.prometheus;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+import java.sql.Timestamp;
+
+@JsonFormat(shape = JsonFormat.Shape.ARRAY)
+public class PrometheusTimeSeriesValue
+{
+    public Timestamp getTimestamp()
+    {
+        return timestamp;
+    }
+
+    public String getValue()
+    {
+        return value;
+    }
+
+    @JsonDeserialize(using = PrometheusTimestampDeserializer.class)
+    private Timestamp timestamp;
+    @JsonProperty
+    private String value;
```

Comment:
Please make these variables to final and move getter to the bottom.

Reply:
Done in 68179648